arelle/ModelValue.py

```diff
@@ -393,20 +468,20 @@ def dayHrsMinsSecs(self):
             seconds += 1
         hours = int(seconds / 86400 )
         if hours > 24:
-            days += hours / 24
+            days = hours / 24 # type: ignore[assignment]
```

Comment:
For some reason, days is a `float` here and I can't seem to change the type.

Reply:
Dropping the `+` appears to be a bug. I think the intent here is to handle scenarios like `days=1, hours=26` and return `days=2, hours=2`. so `days += hours // 24` will get the correct result and give you an int `days` value.

Reply:
Oh, that was a mistake on my end. Probably did it when trying to figure out the float. I'll revert.